be/src/storage/segment_sync_executor.h

```diff
@@ -0,0 +1,135 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Inc.
+#pragma once
+
+#include <atomic>
+#include <memory>
+#include <vector>
+
+#include "common/status.h"
+#include "gen_cpp/doris_internal_service.pb.h"
+#include "gen_cpp/internal_service.pb.h"
+#include "storage/olap_define.h"
+#include "util/reusable_closure.h"
+#include "util/spinlock.h"
+#include "util/threadpool.h"
+
+namespace starrocks {
+
+class DataDir;
+class ExecEnv;
+class SegmentPB;
+class PTabletInfo;
+class FileSystem;
+
+namespace vectorized {
+class DeltaWriterOptions;
+}
+
+using DeltaWriterOptions = starrocks::vectorized::DeltaWriterOptions;
+
+class SyncChannel {
```

Comment:
Sync may be a misunderstanding noun with the sync to disk. I think you can use Replicate to describe it.

Reply:
seems good, i will fix it.